app/models/bulkrax/importer.rb

```diff
@@ -96,16 +96,19 @@ def schedulable?
     end
 
     def current_run
-      @current_run ||= if file? && zip?
-                         self.importer_runs.create!
-                       else
-                         entry_counts = {
-                           total_work_entries: self.limit || parser.works_total,
-                           total_collection_entries: parser.collections_total,
-                           total_file_set_entries: parser.file_sets_total
```

Comment:
Attempting to read the CSV throws the `CSV::MalformedCSVError`, which causes the `ImporterRun` creation to fail, which means that a status cannot be logged. This change adds an additional write to the db, which isn't ideal, but it ensures the `ImporterRun` will always be created successfully